src/modules/70-pipeline/pages/webhooks/utils.tsx

```diff
@@ -49,3 +51,43 @@ export enum WebhookTabIds {
   ListTab = 'ListTab',
   EventsTab = 'EventsTab'
 }
+
+export type WebhookEventStatus = 'FAILED' | 'PROCESSING' | 'QUEUED' | 'SKIPPED' | 'SUCCESSFUL'
+
+export const stringsMap: Readonly<Record<WebhookEventStatus, StringKeys>> = {
+  FAILED: 'pipeline.executionFilters.labels.Failed',
+  PROCESSING: 'pipeline.eventStatus.processing',
+  QUEUED: 'pipeline.eventStatus.queued',
+  SKIPPED: 'pipeline.testsReports.skipped',
+  SUCCESSFUL: 'pipeline.executionFilters.labels.Success'
+}
+
+export function getStatusList(getString: (key: keyof StringsMap) => string): SelectOption[] {
+  return Object.keys(stringsMap).map(status => {
+    return {
+      label: getString(stringsMap[status as WebhookEventStatus]),
+      value: status
+    }
+  })
+}
+
+export const iconMap: Record<WebhookEventStatus, IconProps> = {
+  SUCCESSFUL: { name: 'tick-circle', size: 10 },
+  FAILED: { name: 'warning-sign', size: 9 },
+  PROCESSING: { name: 'loading', size: 10 },
+  QUEUED: { name: 'queued', size: 10 },
+  SKIPPED: { name: 'skipped', size: 8 }
+}
+
+export function gitFilePath(repo: string | undefined, filePath: string | undefined): JSX.Element {
```

Comment:
Not an issue, but try optional instead of `undefined`
```suggestion
export function gitFilePath(repo?: string, filePath?: string): JSX.Element {
```